File: app/src/main/java/com/example/newbiechen/ireader/utils/ToastUtils.java
package com.example.newbiechen.ireader.utils;

import android.widget.Toast;

import com.example.newbiechen.ireader.App;

/**
 * Created by newbiechen on 17-5-11.
 */

public class ToastUtils {

    public static void show(String msg){
        Toast.makeText(App.Companion.getInstance(), msg, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/example/newbiechen/ireader/utils/ToastUtils.kt
package com.example.newbiechen.ireader.utils

import android.widget.Toast
import com.example.newbiechen.ireader.App

object ToastUtils {
    @JvmStatic
    fun show(msg: String) {
        Toast.makeText(App.getInstance(), msg, Toast.LENGTH_SHORT).show()
    }
}